File: online-judge-solutions/Codeforces/1216A.py
#https://codeforces.com/problemset/problem/1216/A
#Lang		:	Python 3.8
#Time		:	171 ms
#Memory         :	1500 KB
n,s=int(input()),input()
r=s
s=list(s)
for i in range(0,n,2):
    na=s[i:i+2].count('a')
    nb=s[i:i+2].count('b')
    if(na!=nb):
        s[i]='a'
        s[i+1]='b'
d=0
s=''.join(s)
for i in range(n):
    if(s[i]!=r[i]):d+=1 
print(d)
print(s)

File: online-judge-solutions/Codeforces/1216A.cpp
//https://codeforces.com/problemset/problem/1216/A
//Lang		:	GNU C++14
//Time		:	46 ms
//Memory	:	500 KB
#include<iostream>
#include<algorithm>
using namespace std;
int main(){
	int n;
	while(cin>>n){
		int d=0;
		string s;
		cin>>s;
		string r=s;
		for(int i=0;i<n;i+=2){
			int na=count(s.begin()+i,s.begin()+i+2,'a');
			int nb=count(s.begin()+i,s.begin()+i+2,'b');
			if(na!=nb){s[i]='a';s[i+1]='b';}
		}
		for(int i=0;i<n;i++){
			if(s[i]!=r[i]) d++;
		}
		cout<<d<<"\n"<<s<<endl;
	}
	return 0;
}
